README.md

```diff
@@ -76,7 +86,7 @@ to rebuild the Docker image for your changes to apply.
 Splits I/O uses [livesplit-core][livesplit-core] for parsing. The parser is located in `lib/parser/*`. To upgrade it,
 run
 ```sh
-docker-compose run web bundle exec rake update_lsc
+make update_lsc
 ```
 and commit the changes.
 
```

Comment:
Can we get a small blurb about the `make clean` recipe, and maybe include that it will destroy the db and local s3?